tox.ini

```diff
@@ -1,95 +1,103 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/zope-product
 [tox]
+minversion = 3.18
 envlist =
-    lint,
-    py27,
-    py27-zserver,
```

Comment:
This is getting rid of tests with ZServer. Intention or oversight?

Reply:
Sorry, this is an oversight, I am going to fix it now.

Reply:
Fixed in https://github.com/zopefoundation/Products.PluggableAuthService/pull/86/commits/09237f2dc49a3f4e96142881b5404c94d1fffa28